### ISSUE TITLE
WebConsole silently drops fields

### ISSUE THREAD

**Issue by lalithsuresh** [feldera/feldera#2363]
I've been trying to configure the pipeline with fields that presumably aren't supported. But when clicking Apply, there is no feedback that those fields are wrong/ignored. Instead, I have to re-open the configuration to see whether it went through.

**Comment by ryzhyk** [comment]
I think part of the problem is the backend not returning an error. Let's make sure that it does, and that the webconsole propagates it to the user. Adding @snkas  to the issue.

**Comment by snkas** [comment]
Yes, there will be a change to using `#[serde(deny_unknown_fields)` everywhere

There is a limitation in which it doesn't allow flatten, so will have to look into that: https://serde.rs/container-attrs.html#deny_unknown_fields

**Comment by snkas** [comment]
This would make backward compatibility very difficult.

**Comment by snkas** [comment]
I think for now this will not be feasible given the above concerns, except for specific cases. Will close for now.
